jobs/auto_complete_sample.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+# [START instantiate]
+from googleapiclient.discovery import build
+
+client_service = build('jobs', 'v2')
+
```

Comment:
You can remove the empty lines between code and region tag.

Reply:
I use `pyformat -i -s 4 --recursive . ` to format my code. And it seems this is recommended. 
Should I manually modify those files? Or it should be fine?